autorest/codegen/models/request_builder.py

```diff
@@ -79,8 +79,13 @@ def imports(self) -> FileImport:
     @classmethod
     def from_yaml(cls, yaml_data: Dict[str, Any], *, code_model) -> "RequestBuilder":
 
+        # when combine embeded builders into one operation file, we need to avoid duplicated build function name.
+        # So add operation group name is effective method
+        additional_mark = ""
+        if code_model.options["combine_operation_files"] and code_model.options["builders_visibility"] == "embedded":
+            additional_mark = "_{}".format(yaml_data["language"]["python"]["builderGroupName"])
         names = [
-            "build",
+            "build{}".format(additional_mark),
```

Comment:
when combine embeded builders into one operation file, we need to avoid duplicated build function name.
So add operation group name is effective method. Relative test: https://github.com/Azure/autorest.python/pull/1012/files#diff-79421dc3d0560d044d44d0eaa0dc8cd500314f3401a0ef85cfca5c815054ed0f

Reply:
done